src/core/lib/debug/stats.h

```diff
@@ -47,23 +47,14 @@ extern Stats* const g_stats_data;
   (&::grpc_core::g_stats_data    \
         ->per_cpu[grpc_core::ExecCtx::Get()->starting_cpu()])
 
-/* Only collect stats if GRPC_COLLECT_STATS is defined or it is a debug build.
- */
-#if defined(GRPC_COLLECT_STATS) || !defined(NDEBUG)
 #define GRPC_STATS_INC_COUNTER(ctr) \
```

Comment:
It seems we are collecting stats in all builds (even non-debug builds) now. Would that lead to any performance impact ?

Reply:
hey yeah... but let's move that comment over to #30610 (that this is based upon)